tests/utils.ts

```diff
@@ -22,6 +31,25 @@ export function generateVarName(prefix = 'v'): string {
   return `${prefix}_${id}`;
 }
 
+/**
+ * Log browser type and version.
+ * @param browser
+ */
+export async function logBrowserInfo(): Promise<void> {
+  const launchers = [chromium, firefox, webkit];
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const launcher of launchers) {
+    // eslint-disable-next-line no-await-in-loop
+    const browser = await launcher.launch();
+
+    // eslint-disable-next-line no-console
+    console.log('Browser:', browser.browserType().name(), browser.version());
+
+    browser.close();
+  }
```

Comment:
Take a look at the description of this rule: https://eslint.org/docs/latest/rules/no-await-in-loop
> However, performing an await as part of each operation is an indication that the program is not taking full advantage of the parallelization benefits of async/await.
> Usually, the code should be refactored to create all the promises at once, then get access to the results using Promise.all(). Otherwise, each successive operation will not start until the previous one has completed.

I think that applies in this case - we should be able to launch all the browsers in parallel and it should be faster (I think?).

In any case, I'd write this something like this instead of disabling the warnings:
```suggestion

  await Promise.all(
    launchers.map(async launcher => {
      const browser = await launcher.launch();
      // eslint-disable-next-line no-console
      console.log('Browser:', browser.browserType().name(), browser.version());

      return browser.close();
    })
  );
```

Reply:
I'll try this in my next PR to avoid having to wait for the e2e tests on this one